server/app.py

```diff
@@ -266,50 +262,80 @@ def player_finished(data):
     if players_finished_set == active_players_set:
         # We also include the winning player name in the 'gameComplete' message.
         winning_player = max(ROOMS[room]['activePlayers'], key=ROOMS[room]['activePlayers'].get)
-        print("THIS IS THE WINNING PLAYER:", winning_player)
-        print("THIS IS THE KEY:", ROOMS[room]['activePlayers'])
         update_db_gameswon(winning_player)
```

Comment:
We need to make sure not to call `update_db_gameswon` if they are playing solo. Checking if `active_players_set` is 1 is enough.

Reply:
Will do once we merge your changes to that function with mine and i will update on the next pull request